content/en/blog/2022-11-11-prove-the-authenticity-of-helm-charts/index.md

```diff
@@ -0,0 +1,276 @@
+tags: [oci, security]
+resources:
+- src: "**.{png}"
+  title: "Image #:counter"
+---
+
+The cosign integration was one of the most important features we shipped in the Flux [v0.35 release](https://github.com/fluxcd/flux2/releases/tag/v0.35.0). After that, we wrote a [blog post](/blog/2022/10/prove-the-authenticity-of-oci-artifacts/) that explains how to use the feature with [OCIRepository](https://fluxcd.io/flux/components/source/ocirepositories/) resources that enables fetching OCI artifacts from container registries, and if you haven't read it yet, we highly encourage you to go and check it out first.
+
+{{< imgproc verify-the-integrity-of-the-helm-charts-stored-as-oci-artifacts-before-reconciling-them-with-flux Resize x700 >}}
+{{< /imgproc >}}
+
+Flux v0.36.0 allows you to prove the authenticity of [HelmChart](/flux/components/source/helmcharts/) resources with the help of the `cosign` integration.  Here we will demonstrate how to use the cosign integration to verify the integrity of the Helm charts stored in OCI-compliant registries as OCI artifacts.
+
+{{< tweet user="stefanprodan" id="1585710554018037761" >}}
+
+Starting with Helm [v3.8.0](https://helm.sh/blog/storing-charts-in-oci/), Helm supports the OCI registry as a one of the storage option for Helm charts as an alternative to Helm repositories. The [Helm CLI](https://helm.sh/docs/helm/helm/) can push and pull Helm charts to and from OCI-compliant registries.
+
+{{% note %}}
+Prior to Helm v3.8.0, OCI support was experimental and must be enabled by setting the `HELM_EXPERIMENTAL_OCI` environment variable to `1`.
+{{% /note %}}
+
+As we store Helm charts in OCI-compliant registries as OCI artifacts, we cab now use the cosign integration to sign and verify them. Also, thanks to Flux, you can reconcile resources such as plain-text Kubernetes YAML manifests, Terraform modules, etc. from OCI-compliant registries with the help of `OCIRepository` resources. You can achieve the same thing for Helm charts with [HelmRepository](/flux/components/source/helmrepositories/#helm-oci-repository) resources, which means that you can store Helm charts in OCI-compliant registries as OCI artifacts and use Flux to reconcile them like the following:
+
+```yaml
+---
+apiVersion: source.toolkit.fluxcd.io/v1beta2
+kind: HelmRepository
+metadata:
+  name: podinfo
+  namespace: default
+spec:
+  type: "oci"
+  interval: 5m0s
+  url: oci://ghcr.io/stefanprodan/charts
+```
+
+> ⚠️ Note: Here you can review the complete registries lists that support the OCI artifact specification: [OCI-Conformant Products](https://conformance.opencontainers.org/#distribution-spec)
+>
+> ⚠️ You will notice that when you open the list, DockerHub is not included into the list but it will be added soon because they recently announced OCI Artifacts support, and you can read more about it from [here](https://www.docker.com/blog/announcing-docker-hub-oci-artifacts-support/).
```

Comment:
You could use `{{% note %}}` for these entries too, so you don't have to show a `warning` sign?